build-system/tasks/dist.js

```diff
@@ -55,6 +60,37 @@ const {maybeUpdatePackages} = require('./update-packages');
 const {green, cyan} = colors;
 const argv = require('minimist')(process.argv.slice(2));
 
+const babel = require('@babel/core');
+const deglob = require('globs-to-files');
+
+function transferSrcsToTempDir() {
+  const tmp = `${tempy.directory()}/amphtml`;
+  process.env.AMP_TMP_DIR = tmp;
+  log(green(`creating temp directory: ${tmp}`));
+
+  if (!isTravisBuild()) {
+    log('Writing transforms to', colors.cyan(tmp));
+  }
+  const files = deglob.sync(BABEL_SRC_GLOBS);
+  log(green('Executing babel transforms'));
```

Comment:
Nit: Prefer this logging style, used elsewhere in `build-system`:

```js
 if (!isTravisBuild()) { 
   log('Transforming JS files with', cyan('babelify'), 'on port', cyan(port)); 
 }
```

For example: 
https://github.com/ampproject/amphtml/blob/c4a663d0ba13d0488c6fe73c55dc8c971ac6ec0d/build-system/tasks/nailgun.js#L105-L107

Reply:
done.